### PR TITLE
Add hypo tool

### PR DESCRIPTION
**Pull request by jesgomez**

### CODE REVIEW

**Comment by bgruening** [comment]
it should go to the tool_iuc.yml file, since its a IUC tool.

File: bgruening.yaml [hunk 1]
```diff
@@ -232,3 +232,6 @@ tools:
     owner: bgruening
     tool_panel_section_label: Single-cell
 
+ - name: hypo
+   owner: bgruening
```
Comment (bgruening) [inline bgruening.yaml:236]: and this should be `iuc`

Comment (jesgomez) [reply bgruening.yaml]: yes, that makes sense. I just fixed it. Thanks

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Cool, thanks a lot. In 30 or so it will be installed!

**Pull request merged**